README.md

```diff
@@ -38,6 +38,8 @@ group :development do
 end
 ```
 
+Note: If using an addon such as `ruby-lsp-rails`, `ruby-lsp` will be declared in the main `Gemfile.lock`, so the `.ruby-lsp/Gemfile` and its lockfile are not used, and can be deleted.
```

Comment:
In the case of the Slack conversation, I suspect they first started with Ruby LSP only, then later added `ruby-lsp-rspec`, which resulted in a redundant `.ruby-lsp/` directory.

Reply:
This is true, I don't remember the exact order of operations, but I know we started with LSP only first. Then we found out the recommended way was to remove it from our Gemfile because it would be managed internally.

I don't remember when exactly we added `ruby-lsp-rspec` but I could dig up the commit if it's helpful.